src/cascadia/QueryExtension/ExtensionPalette.cpp

```diff
@@ -35,13 +35,13 @@ namespace winrt::Microsoft::Terminal::Query::Extension::implementation
         ControlName(RS_(L"ControlName"));
         QueryBoxPlaceholderText(RS_(L"CurrentShell"));
 
-        auto disclaimerLinkText = Windows::UI::Xaml::Documents::Run();
-        disclaimerLinkText.Text(RS_(L"AIContentDisclaimerHyperlink"));
-        AIContentDisclaimerHyperlink().Inlines().Append(disclaimerLinkText);
+        std::unordered_map<size_t, std::wstring> disclaimerPlaceholderToStringMap;
```

Comment:
why's this a map? placeholders are linear... can we get away with a `std::span`?